src/main/java/com/hibernate/cinema/model/Ticket.java

```diff
@@ -0,0 +1,44 @@
+package com.hibernate.cinema.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "tickets")
+public class Ticket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne
+    private MovieSession movieSession;
+    @OneToOne
```

Comment:
```suggestion
    @ManyToOne
```
Similarly, one user can have many tickets, I think